### PR TITLE
[bitnami/mysql] add parameter to set up my_custom.cnf

### LINKED ISSUE #6423: [bitnami/mysql] Write your my_custom.cnf file with the following content.
**Which chart**:
bitnami/mysql 

**Is your feature request related to a problem? Please describe.**
I need add to replica some additional config 
```replicate-ignore-table=logtable```
or on master 
```
[mysqld]
max_allowed_packet=32M
```
like describe 
https://github.com/bitnami/bitnami-docker-mysql#step-1-write-your-my_customcnf-file-with-the-following-content

### PR DESCRIPTION
**Pull request by serg-bs**
**Description of the change**
I need add to replica some additional config 
```replicate-ignore-table=logtable```
or on master 
```
[mysqld]
max_allowed_packet=32M
```
like describe 
https://github.com/bitnami/bitnami-docker-mysql#step-1-write-your-my_customcnf-file-with-the-following-content

**Applicable issues**

  - fixes #6423

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

**Comment by alemorcuq** [comment]
Hi, @serg-bs. Thanks for the PR. I can see the chart already has two configuration sections: `primary.configuration`, where you can set all the configuration options you want in your `my.cnf` file, and a `primary.existingConfiguration` value where you can set a ConfigMap that holds your full configuration. They are also available for the secondary pods. Isn't this enough for you?

**Comment by serg-bs** [comment]
Yes this section is optional, in my case developer don't want to know about all configuration esspecialy about directory structure. In postgres chart we have postgresqlExtendedConf, and it look quite convinient.

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Could you also add support for an existing ConfigMap like in the PostgreSQL chart?

File: bitnami/mysql/templates/secondary/configmap.yaml [hunk 1]
```diff
@@ -13,6 +13,6 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  my.cnf: |-
+  my.cnf: |+
```
[the remark this answers is not in the document]
Comment (serg-bs) [reply bitnami/mysql/templates/secondary/configmap.yaml:16]: That is most important changes. Whithout this changes config not work.
trailing newline is important in my.cnf
`!include /opt/bitnami/mysql/conf/bitnami/my_custom.cnf` after this one should be newline

**Pull request closed without merging**